upstream/manifests/v3.5.1/sriovdp-daemonset.yaml

```diff
@@ -0,0 +1,223 @@
+# Pulled from
+# https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/raw/v3.5.1/deployments/k8s-v1.16/sriovdp-daemonset.yaml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sriov-device-plugin
+  namespace: kube-system
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-sriov-device-plugin-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: sriovdp
+spec:
+  selector:
+    matchLabels:
+      name: sriov-device-plugin
+  template:
+    metadata:
+      labels:
+        name: sriov-device-plugin
+        tier: node
+        app: sriovdp
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: sriov-device-plugin
+      containers:
+      - name: kube-sriovdp
+        image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin:v3.5.1
```

Comment:
did these images get copied into rocks?

Reply:
Yep, I sync'd it with regsync. Hopefully this is convincing?

```
$ docker manifest inspect ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin:v3.5.1 > upstream
$ docker manifest inspect rocks.canonical.com/cdk/k8snetworkplumbingwg/sriov-network-device-plugin:v3.5.1 > rocks
$ diff upstream rocks; echo $?
0
```